hack/generator/pkg/astbuilder/calls.go

```diff
@@ -69,5 +64,5 @@ func InvokeQualifiedFunc(qualifier *ast.Ident, funcName *ast.Ident, arguments ..
 // <qualifier>.<funcName>(arguments...)
 // If you want to use the result of the function call as a value, use CallQualifiedFuncByName() instead
 func InvokeQualifiedFuncByName(qualifier string, funcName string, arguments ...ast.Expr) ast.Stmt {
-	return InvokeQualifiedFunc(ast.NewIdent(qualifier), ast.NewIdent(funcName), arguments...)
+	return InvokeQualifiedFunc(qualifier, funcName, arguments...)
```

Comment:
Now that the parameters are simply being passed through without change, this method should be deleted.

Reply:
it has gone